web/src/composebox_typeahead.ts

```diff
@@ -936,17 +924,48 @@ export function get_candidates(
                 }
 
                 const stream_id = stream_data.get_stream_id(stream_name);
+                // We always show topic suggestions after the user types a stream,
+                // and let them pick between just showing the stream (the first option, when nothing follows ">")
+                // or adding a topic.
                 const topic_list = topics_seen_for(stream_id);
+
                 if (should_show_custom_query(token, topic_list)) {
                     topic_list.push(token);
                 }
                 const matcher = get_topic_matcher(token);
                 const matches = topic_list.filter((item) => matcher(item));
+                const stream = stream_data.get_sub_by_name(stream_name);
+                assert(stream !== undefined);
```

Comment:
This lookup should probably be by `stream_id`, since that's more robust to streams being renamed. Or perhaps a single lookup of this form should occur above where we get `stream_id`, and code needing `stream_id` should be accessing `stream.id`. Also, we usually use `sub` as the local variable for a StreamSubscription object. 

So I think we want a `const sub = stream_data.get_sub_by_name(stream_name();` about 10 lines up, immediately followed by this assert, and then working with that `sub` object.